includes/apple-exporter/components/class-cover.php

```diff
@@ -172,6 +172,18 @@ protected function build( $options ) {
 			return;
 		}
 
+		// Use postmeta to determine if component role should be registered as 'image' or 'photo'.
+		$use_image  = get_post_meta( $this->workspace->content_id, 'apple_news_use_image_component', true );
+		$image_type = $use_image ? 'image' : 'photo';
+
+		/**
+		 * Allows for an image src value to be filtered before being applied.
+		 *
+		 * @param string $image_type The image component type as determined by postmeta ('image' or 'photo').
+		 * @param string $options    The options based to the cover's build method.
+		 */
+		$role = apply_filters( 'apple_news_build_cover_img_role', $image_type, $options );
```

Comment:
The more I think about it, I think we shouldn't provide a filter here. Users can filter the JSON for a component generally and could modify the role if they chose. Additionally, we'd need to provide filters for both the cover and the image component, since users might want different behavior in each component, and the arguments would be different for each, at which point they should just use the component filter.